ams/ams-server/src/test/java/JacksonTest.java

```diff
@@ -0,0 +1,121 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.netease.arctic.ams.server.config.ConfigFileProperties;
+import com.netease.arctic.ams.server.utils.JacksonUtils;
+import org.junit.Test;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.*;
+
+public class JacksonTest {
+
+
+
+    @Test
+    public void jacksonTest(){
+        String configPath = System.getProperty("user.dir") + "/src/test/resources/test.yaml";
+        JsonNode yamlConfig = load(configPath);
+        ObjectNode systemConfig =(ObjectNode) yamlConfig.get(ConfigFileProperties.SYSTEM_CONFIG);
+        systemConfig.put("test", "test1");
+        LinkedHashMap<String, Object> config = new LinkedHashMap<String, Object>();
+        config.putAll(JacksonUtils.parseObject(JacksonUtils.toJSONString(systemConfig), Map.class));
+        JsonNode catalogs = yamlConfig.findValues(ConfigFileProperties.CONTAINER_LIST).get(0);
+        for(JsonNode jsonNode : catalogs){
+            System.out.println(jsonNode.get("name").asText());
+        }
+        System.out.println(systemConfig.has("arctic.ams.expire.thread.pool-size"));
```

Comment:
We cannot simply output at this point, we need to assert the correctness of the value. You can refer to other unit tests for how to do this.